app/controllers/application_controller.rb

```diff
@@ -1,23 +1,21 @@
 class ApplicationController < ActionController::Base
   include ScramUtils
   helper_method :current_holder
+  helper_method :current_team
 
   protect_from_forgery with: :exception
-  before_action :set_team
+
+  def current_team
+    current_user.try(:current_team)
```

Comment:
^ need to cache this per-request, since User#current_team creates queries in Redis _and_ the DB.